local-source-proxy/design.hbs.md

```diff
@@ -7,15 +7,17 @@ registry server, which is configured through the `tap-values.yaml` file.
 Local Source Proxy takes care of authentication and authorization against the external registry
 provider internally. This ensures that the external registry remains transparent to the user.
 
-A Kubernetes service is set up to access Local Source Proxy, and it can be reached by using an
-internal URL within the Kubernetes cluster.
-<!-- what is the it? -->
-
 By functioning as a proxy registry server, Local Source Proxy simplifies the process of
 interacting with external registry servers. Local Source Proxy provides a centralized and transparent
 approach for image push requests, handling authentication and authorization seamlessly.
 
-![Diagram showing the relationship between an external registry, a developer workstation, and a Kubernetes cluster.](images/lsp-design.png)
+LSP Active
```

Comment:
Can we rename the tabs to more Human readable format? Like
`Using Local Source Proxy` and `Using --source-image Flag`